save-orchestrator-common/src/test/kotlin/com/saveourtool/save/orchestrator/controller/heartbeat/HeartbeatControllerTest.kt

```diff
@@ -68,12 +69,12 @@ class HeartbeatControllerTest {
             .mutate()
             .responseTimeout(2.seconds.toJavaDuration())
             .build()
+        agentStatusInMemoryRepository.clear()
     }
 
     @AfterEach
     fun cleanup() {
         verifyNoMoreInteractions(orchestratorAgentService)
-        agentStatusInMemoryRepository.clear()
```

Comment:
don't we need to hold this anyway? what if this `cleanup` will be called manually and expect the cleaning?